virtulos/FCO
# Implement Unit tests for the FCO contracts

### How to run tests
Once you installed dependencies, you can run the Unit tests via hardhat test with command `yarn test` or `npm test`

The tests also consider adding a lot of slots of locked tokens for the user (1000) to make sure user can claim his tokens in several years without issues.


### Test results

<img width="798" alt="image" src="https://github.com/virtulos/FCO/assets/32820910/1168c15d-8830-4344-b779-54cf18d3b7e1">
